CHANGELOG.md

```diff
@@ -1,6 +1,47 @@
 Change log
 ==========
 
+1.5.0 (2015-10-13)
+------------------
+
+Major Features
+
+-   Compose is now available on windows.
+-   Environment variable can be used in the compose file. See
+    https://github.com/docker/compose/blob/master/docs/yml.md#variable-substitution
```

Comment:
Good point :/

Perhaps leave it like this during the RC period, and update together with the bump?
Is there a tag created for the RC's?